### PR TITLE
alpine improvements

### PR DESCRIPTION
**Pull request by its-pointless**
on a fresh install creates a passwordless master password so its less annoying for users on mobiles to reenter pass constantly.

### CODE REVIEW

File: packages/alpine/build.sh [hunk 1]
```diff
@@ -41,4 +42,14 @@ termux_step_post_configure() {
 	$CC_FOR_BUILD help_h_gen.c -o help_h_gen
 	touch -d "next hour" help_c_gen help_h_gen
 }
+termux_step_create_debscripts () {
+
+        echo "#!$TERMUX_PREFIX/bin/sh" >> postinst
+	echo "if [ ! -e $TERMUX_ANDROID_HOME/.alpine-smime ] && [ ! -e $TERMUX_ANDROID_HOME/.pine-passfile ]; then" >> postinst
```
Comment (rrthomas) [inline packages/alpine/build.sh]: I'm not sure it's necessary to test for existing `.pine-passfile`: the user might have copied this already. I would say it's only necessary to check they don't already have a master password.

**Comment by rrthomas** [comment]
I made one small comment; otherwise, looks good, thanks!

**Comment by rrthomas** [comment]
Looks fine now; thanks very much for working on this!

**Comment by rrthomas** [comment]
What is required to get this PR merged and a package built? Anything I can do to help? (I just got an update from the repo to version 2.21, but sadly without these patches!)

**Comment by its-pointless** [comment]
@fornwall ?

**Comment by fornwall** [comment]
Thanks - the updated package is now available for installation!

**Pull request merged**